v5/pivotal/aggregator.go

```diff
@@ -0,0 +1,224 @@
+package pivotal
+
+import (
+	"encoding/json"
+	"fmt"
+	"math"
+)
+
+const aggregatorURL = "https://www.pivotaltracker.com/services/v5/aggregator"
+
+// PT doesn't allow more than around 15 urls in one aggregation request.
+// However, this was found by experimenting. There was no official
+// rate-limiting data in their docs.
+const perPage = 15 // GET requests per aggregation requests.
+
+// AggregatorService is used to wrap the client.
+type AggregatorService struct {
+	client *Client
+}
+
+type AggregationRequest struct {
+	url       string
+	projectID int
+	storyID   int
+}
+
+// Aggregation object stores the state of the aggregation.
+type Aggregation struct {
+	// Requests are the GET requests that are used by the aggregator.
+	requests []string
+
+	// Storing the response json along with the urls.
+	aggregatedResponse map[string]interface{}
+
+	// The service of the aggregation
+	service *AggregatorService
+}
+
+func (a *AggregatorService) GetBuilder() *Aggregation {
+	aggregation := Aggregation{
+		aggregatedResponse: make(map[string]interface{}),
+		service:            a,
+	}
+	return &aggregation
+}
+
+func newAggregatorService(client *Client) *AggregatorService {
+	return &AggregatorService{client}
+}
+
+// Adds the url for the story to the aggregation data.
```

Comment:
It is idiomatic to start the documentation with the name of the thing you are documenting , e.g., this might be "Story adds the url for the story to the aggregation data."

Would you be willing to update this for all the public member in this PR?